bimmer_connected/cli.py

```diff
@@ -46,6 +46,11 @@ def main_parser() -> argparse.ArgumentParser:
     flash_parser.add_argument("vin", help=TEXT_VIN)
     flash_parser.set_defaults(func=light_flash)
 
+    horn_parser = subparsers.add_parser("horn", description="Flash the vehicle lights.")
```

Comment:
Please also update the description to `Sound the horn.`. Sorry, missed that one earlier as well.